slang.md

```diff
@@ -13,3 +13,9 @@ Former functional human beings that suffered from the Nueroinfinitus (forevermor
 ### Forevermore Plague
 
 Nueroinfinitus also known as the Forevermore Plague was a widedspread virus that effected nearly all persons who had cyber implants
+
+### The Road
+The YANG LIWEI INTERNATIONAL SPACE LIFT. nothing on earth has caused a larger technological, scientific, economic, or industrial boom. Some say "The Road" is the apex achievement of humankind.
```

Comment:
I like the bold title for the space lift, but should we format this differently?

Reply:
i just copied and pasted from roll 20-  yeah it should be lowercase

Reply:
yep! i just copy/pasted from roll 20 it should be consistent here ill make lowercase

Reply:
@The-Real-Matthew-Dycus  I will modify to lower case.